vm.go

```diff
@@ -241,6 +243,19 @@ func (vm *VirtualMachineSpec) Delete() error {
 	return system.Terminate(ctx)
 }
 
+// Wait for a Virtual Machine exits
+func (vm *VirtualMachineSpec) Wait() error {
```

Comment:
I'll approve, but where does this get used?

Reply:
It will be used in node agent: https://github.com/microsoft/wssdagent/pull/437